clitests/tests/encode/encode_astc_bad_format.json

```diff
@@ -0,0 +1,258 @@
+{
+    "description": "Test encode command with unsupported formats.",
+    "command": "ktx encode --testrun --threads 1 --format ${format} input/ktx2/valid_known_R8G8B8_UNORM_2D.ktx2 output/encode/encode_error_format/output_${format}.ktx2",
+    "status": 1,
+    "stderr": "golden/encode/encode_error_format/common_format.err.txt",
+    "args": {
+        "format": [
+            "R4G4_UNORM_PACK8",
```

Comment:
I think we should include some ASTC formats here to make sure people can't decode and re-encode differently.

Actually a different error message would be good in this case. Something like "already encoded as ASTC.". So testing those formats will likely be a different test.

Reply:
This test is checking for invalid `--format`s to `ktx encode` in particular. ASTC format to `--format` in not invalid. As you said checking existing format already ASTC is a different error and also requires different functionality first. I shall add that first.

Reply:
Okay. Add the ASTC tests in another test.